pkg/ansible/proxy/proxy.go

```diff
@@ -121,9 +195,30 @@ func Run(done chan error, o Options) error {
 		server.Handler = o.Handler(server.Handler)
 	}
 
+	if o.Cache == nil {
+		// Need to initialize cache since we don't have one
+		log.Info("Initializing and starting informer cache...")
+		informerCache, err := cache.New(o.KubeConfig, cache.Options{})
+		if err != nil {
+			return err
+		}
+		stop := make(chan struct{})
+		go func() {
+			informerCache.Start(stop)
+			defer close(stop)
+		}()
+		log.Info("Waiting for cache to sync...")
```

Comment:
I suggest adding a log message after the sync to confirm that it finished. A user who sees this in the log without any other mention may get the impression that it never completed.